lib/rucio/transfertool/cms_fts3.py

```diff
@@ -0,0 +1,358 @@
+# Copyright European Organization for Nuclear Research (CERN)
```

Comment:
I wrote the script ./tools/generate_header to get the header decided within ATLAS and the right apache v2 text. I would propose that you use it for now..

Reply:
@vingar actually I just copied the transfertool/fts3.py one and I cannot find the generate_header script. Can you send me the link for that?

Reply:
it's in the repo `./tools/generate_header`

`./tools/generate_header  <file>`

Reply:
yes, that's correct.